backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickhouseFunctionSuite.scala

```diff
@@ -137,4 +140,59 @@ class GlutenClickhouseFunctionSuite extends GlutenClickHouseTPCHAbstractSuite {
       assert(diffCount == 0)
     }
   }
+
+  private def checkFallbackOperators(df: DataFrame, num: Int): Unit = {
```

Comment:
it's better to put this tool function into WholeStageTransformerSuite ?

Reply:
moved